tests/dependencies/indirect/test

```diff
@@ -3,15 +3,15 @@
 
 # used_deps off
 ! bazel build :used_deps_off_c || false
-bazel build :used_deps_off_c |& grep "buildozer 'add deps //dependencies/indirect:used_deps_off_a' //dependencies/indirect:used_deps_off_c"
+bazel build :used_deps_off_c |& grep "buildozer 'add deps @\?//dependencies/indirect:used_deps_off_a' //dependencies/indirect:used_deps_off_c"
```

Comment:
In Bazel 6 the `@` exists while it doesn't in Bazel 5, therefore the optional check here. I have verified that buildozer handles the `@` correctly and will not add it to the `BUILD.bazel` file (it adds the dep without the `@`)